ymcatwincities/openy
# [carnation] Fix for openy subcategory page image resize issue.

Fixes: https://github.com/ymcatwincities/openy/issues/2539

Steps to test:
- Visit an openy site.
- Go to the program subcategory page preferably with category listing block present in that.
- Now, adjust the content for the program subcategories that displayed in that page and reduce the description to two or three words and save the content. Next, refresh the page.
- Adjust the browser to different breakpoints for mobile devices to check the fix for the issue.
